torch/testing/_internal/common_utils.py

```diff
@@ -1227,6 +1227,9 @@ def remove_prefix(text, prefix):
         def accept_output(update_type):
             print("Accepting {} for {}{}:\n\n{}".format(update_type, munged_id, subname_output, s))
             with open(expected_file, 'w') as f:
+                # Adjust for producer_version
+                s = re.sub(r'(producer_version): "[0-9.]*"',
+                           r'\1producer_version: "CURRENT_VERSION"', s)
```

Comment:
This is the change needed to replace the version i.e., "1.7" with "CURRENT_VERSION" in the saved files.